bin/prettier.js

```diff
@@ -9,25 +9,8 @@ const minimist = require("minimist");
 const prettier = require("../index");
 
 const argv = minimist(process.argv.slice(2), {
-  boolean: [
-    "write",
-    "stdin",
-    "single-quote",
-    "trailing-comma",
-    "bracket-spacing",
-    // The supports-color package (a sub sub dependency) looks directly at
-    // `process.argv` for `--no-color` and such-like options. The reason it is
-    // listed here is to avoid "Ignored unknown option: --no-color" warnings.
-    // See https://github.com/chalk/supports-color/#info for more information.
-    "color",
-    "help",
-    "version",
-    "debug-print-doc",
-    "debug-check",
-    // Deprecated in 0.0.10
-    "flow-parser"
-  ],
-  string: ["print-width", "tab-width", "parser"],
+  boolean: true,
```

Comment:
@jlongster @lydell I think leaving the generic `--trailing-commas` and either add a `--stable` or an `--experimental` flag could do the trick, and would also cater for any other issue that crops up like this one. You can just put currently unsupported syntax behind this flag. It's also less maintenance on users, as they can eventually just upgrade the package (and node I suppose) once it is supported and not need to remember to change the command.